pkg/controller/operators/operator_controller.go

```
@@ -144,11 +150,19 @@ func (r *OperatorReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 
 	}
 
-	if err := r.Status().Update(ctx, operator.Operator); err != nil {
-		log.Error(err, "Could not update Operator status")
-		return ctrl.Result{}, err
+	if create {
+		if err := r.Create(context.Background(), operator); err != nil && !apierrors.IsAlreadyExists(err) {
+			r.log.Error(err, "Could not create Operator", "operator", name)
```

Is there a reason we don't want to return an error here and requeue? Any error at this point should be transient, and the work queue should rate limit to prevent hot loops.

---

Yes, we do want to requeue.  Updated.